src/controllers/user/artists/routes/{id}/index.js

```diff
@@ -157,7 +157,7 @@ module.exports = function () {
 
   async function GET (ctx, next) {
     const where = {
-      creator_id: ctx.profile.id,
+      // creator_id: ctx.profile.id,
```

Comment:
```suggestion
      ownerId: ctx.profile.id,
```

We wanna make sure it only finds artists that actually belong to the user no?

Reply:
I'll take your word for it. Poked around with this part until it returned some data, instead of errors. 

Still getting my head around this part of the code.

Reply:
And it's broken (and just largely copy+pasted) code at that so I understand that it can be frustrating!